x-pack/plugins/alerting/server/task_runner/inject_action_params.ts

```diff
@@ -36,6 +36,32 @@ export function injectActionParams({
     };
   }
 
+  if (actionTypeId === '.pagerduty') {
+    /**
+     * TODO: Remove and use connector adapters
+     */
+    const path = ruleUrl?.absoluteUrl ?? '';
+
+    if (path.length === 0) {
+      return actionParams;
+    }
+
+    const links = (actionParams.links ?? []) as Array<{ href: string; text: string }>;
+
+    return {
+      ...actionParams,
+      links: [
+        {
+          href: path,
+          text: i18n.translate('xpack.alerting.injectActionParams.pagerduty.kibanaLinkText', {
+            defaultMessage: 'Elastic Rule',
```

Comment:
It might be useful to add the name, id and space (if not default space) of the rule.

Reply:
@pmuellr I added the rule name. Pagerduty shows the links as

<img width="494" alt="Screenshot 2023-11-07 at 6 17 38 PM" src="https://github.com/elastic/kibana/assets/7871006/c9e3de77-2530-4574-b4ba-8995127a36b1">

It felt a bit weird to include the ID and the space as the text can be very long. Wdyt?

Reply:
id can definitely be left out - especially since it will be in the link

I think it's fine to not include the space as well.  Perhaps we'll get some feedback that folks would want it.  But that info will also be in the link :-)

Reply:
Thanks @pmuellr. Then I will only pass the `rule.name` to the `injectActionParams` to avoid any accidental mutation.